ui/src/components/ValidationMessage.vue

```diff
@@ -0,0 +1,26 @@
+<template>
+    <div v-if="model.$invalid" class="p-mtb">
+        <li class="p-error" v-for="err in model.$errors" :key="err">{{ err.$message }}</li>
```

Comment:
Yes. We can use <li> tag with any other DOM element (<div>) without <ul> OR <ol>. 
Now, I changed this to <p> tag as suggested. 
Note : <span> tag not used because custom CSS required to add on it whether <p> provide built-in CSS for element.